doc/introduction/chemistry.rst

```diff
@@ -43,7 +43,10 @@ installation. This backend is non-differentiable and can be selected by setting
 ``method='pyscf'`` in :func:`~.molecular_hamiltonian`. Additionally, if the electronic Hamiltonian
 is built independently using `OpenFermion <https://github.com/quantumlib/OpenFermion>`_ tools, it
 can be readily converted to a PennyLane observable using the
-:func:`~.pennylane.import_operator` function.
+:func:`~.pennylane.import_operator` function. There is also capability to import wavefunctions (states) 
```

Comment:
Could we make this and the Hamiltonian part a new section?
Something like "Importing the Hamiltonian and ansatz state".
This section could be moved below lines 51 - 67.

Reply:
Sounds good, will do!

Reply:
Done.